From: Lennart Weller <lhw@ring0.de>
Date: Tue, 12 Jun 2018 12:06:59 +0200
Subject: Makefile changes for debian packaging

 - Remove old libssl linking
 - Change build_version to debian package version for easier debugging
 - Include debian package buildflags
---
 client/Makefile | 11 +++++++----
 1 file changed, 7 insertions(+), 4 deletions(-)

diff --git a/client/Makefile b/client/Makefile
index 3c0f741..0fba19e 100644
--- a/client/Makefile
+++ b/client/Makefile
@@ -1,13 +1,16 @@
 BINARY  = looking-glass-client
-CFLAGS  = -g -O3 -std=gnu99 -march=native -Wall -Werror -I./ -I../common -DDEBUG -DATOMIC_LOCKING
-LDFLAGS = -lrt
+
+CFLAGS  += $(CPPFLAGS)
+
+CFLAGS  += -march=native -std=gnu99 -I./ -I../common -DDEBUG -DATOMIC_LOCKING
+LDFLAGS += -lrt
 
 CFLAGS  += -ffast-math
 CFLAGS  += -fdata-sections -ffunction-sections
 LDFLAGS += -Wl,--gc-sections
 CFLAGS  += -Wfatal-errors
 
-LIBS    = sdl2 SDL2_ttf gl glu libssl openssl spice-protocol fontconfig x11 libconfig
+LIBS    = sdl2 SDL2_ttf gl glu spice-protocol fontconfig x11 libconfig
 
 LIBS    += nettle hogweed
 CFLAGS  += -D USE_NETTLE
@@ -21,7 +24,7 @@ LDFLAGS += $(shell pkg-config --libs   $(LIBS))
 BUILD   ?= .build
 BIN     ?= bin
 
-CFLAGS  += -DBUILD_VERSION='"$(shell git describe --always --long --dirty --abbrev=10 --tags)"'
+CFLAGS  += -DBUILD_VERSION='"debian/$(DEB_VERSION)"'
 
 OBJS	= main.o \
 	  lg-renderer.o \
